src/pages/account/profile/user-profile/update.tsx

```diff
@@ -23,13 +23,11 @@ import { sentryMiddleware } from "@src/middlewares/sentry-middleware";
 const UpdateUserProfilePage: React.FC = () => {
```

Comment:
Following the recent discussion, the route should be `src/pages/account/profile/user-profile/edit.tsx`, only the form should be named `Update`.

Reply:
I made it, but I had conflicts with your last PR merged, and I assumed it was still `update` since on your branch it hadn't changed, and I undid it. I will change it back then.

Reply:
Sorry for that. I didn't wanted to make any changes on the file as you were working on it. Quite tricky to all work on the same parts of the app at the same time :/.